sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -89,6 +111,10 @@ def get_media_settings_key(physical_port, transceiver_dict, port_speed, lane_cou
             media_compliance_dict = ast.literal_eval(media_compliance_dict_str)
             if sup_compliance_str in media_compliance_dict:
                 media_compliance_code = media_compliance_dict[sup_compliance_str]
+                # For 100G transceivers, it's usually in extended specification compliance
+                if media_compliance_code in sup_compliance_extended_values and \
+                        extended_spec_compliance_str in media_compliance_dict:
+                    media_compliance_code = media_compliance_dict[extended_spec_compliance_str]
```

Comment:
Could you provide an example for the media_key generated for a module with the following entry (from the PR description):
`Extended Specification Compliance: 100GBASE-CR4, 25GBASE-CR CA-25G-L or 50GBASE-CR2 with RS`

Will the media_key include all three options listed here, or only the one relevant to the module's speed and lane count? If the entire string is included, the key might end up very long.